models/_sector/nft/trades/chains/base/platforms/magiceden_base_base_trades.sql

```diff
@@ -141,7 +141,7 @@ WITH trades AS (
     , MAX_BY(tr.to, tr.amount_raw) FILTER (WHERE to != 0xca9337244b5f04cb946391bc8b8a980e988f9a6a) AS royalty_fee_address
     , SUM(tr.amount_raw) FILTER (WHERE to = 0xca9337244b5f04cb946391bc8b8a980e988f9a6a) AS platform_fee_amount_raw
     , SUM(tr.amount_raw) FILTER (WHERE to != 0xca9337244b5f04cb946391bc8b8a980e988f9a6a) AS royalty_fee_amount_raw
-    FROM {{ ref('tokens_base_transfers') }} tr
+    FROM {{ ref('tokens_base_base_transfers') }} tr
```

Comment:
this will end up being a source when we move, yes? i'm fine with no immediate action, but worth preparing for.

Reply:
yes, basically this list of models will need to become sources in the NFT sub-project.
(this is only for `nft.trades`, the other nft spells will have more non-nft upstream models)
```
spellbook.prices.prices_usd_forward_fill
spellbook.prices.prices_usd_latest
spellbook._sector.tokens.base.tokens_base_base_transfers
spellbook.tokens.tokens_nft
spellbook._sector.tokens.optimism.tokens_optimism_base_transfers
spellbook._sector.tokens.zksync.tokens_zksync_base_transfers
```